src/auth_service/user_management/claims_repository/core/seed.py

```diff
@@ -61,14 +59,12 @@ async def _remove_existing_data_steward_claims(claim_dao: ClaimDao) -> None:
     log.info("Removed %d existing data steward claim(s).", num_removed_claims)
 
 
-async def _add_user_with_ext_id(user: str | dict, user_dao: UserDao) -> User:
+async def _add_user_with_ext_id(info: DataStewardInfo, user_dao: UserDao) -> User:
```

Comment:
Actually this function _does_ just add a user to the system. I think the confusing bit is that it takes a `DataStewardInfo` even though this info can describe any user. This was meant to make the config more clear, but I think it is sufficient to document it on the level of the list of data stewards instead of on every item. Renamed this to `UserWithIVA` in 1492b1a6.

Reply:
That makes more sense, thanks for the clarification.